BoxContentSDK/BoxContentSDK/Categories/UIApplication+ExtensionSafeAdditions.m

```diff
@@ -12,18 +12,23 @@ + (UIApplication *)box_sharedApplication
 {
 #ifndef BOX_TARGET_IS_EXTENSION
```

Comment:
Do we want to keep the conditional compiling? If it's a dylib from a pod, it will only get compiled once for all targets including extensions, no?